website/content/en/docs/installation/_index.md

```diff
@@ -36,7 +36,7 @@ export OS=$(uname | awk '{print tolower($0)}')
 Download the binary for your platform:
 
 ```sh
-export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.7.1
+export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.8.0
```

Comment:
:+1:

Reply:
The update script needed to be fixed, so assumed this would auto-generate a wrong link, but the link seems good. :)